ShaderConverter/ShaderConv/context.cpp

```diff
@@ -1187,19 +1215,33 @@ CContext::Translate_M3x2( const CInstr& instr )
 void
 CContext::Translate_POW( const CInstr& instr )
 {
-    // log s0,  abs( src0 )
-    // mul s0,  s0, src1
-    // exp dest, s0
+    // log  s0.y,  abs( src0 )
+    // mov  s0.x, abs( src0 )
+    // movc s0, s0.x, s0.y, vec4( -FLT_MAX, -FLT_MAX, -FLT_MAX, -FLT_MAX )

```

Comment:
[src0 and src1 can be scalar, vector, or matrix](https://learn.microsoft.com/en-us/windows/win32/direct3dhlsl/dx-graphics-hlsl-pow), so I think this needs to be more along the lines of

```
log  s0,  abs( src0 )
mov  s1, abs( src0 )
movc s0, s1, s0, vec4( -FLT_MAX, -FLT_MAX, -FLT_MAX, -FLT_MAX )
```